tests/tests.js

```diff
@@ -0,0 +1,693 @@
+var chai = require('chai');
+var usngs = require('../usng');
+var converter = new usngs.Converter();
+
```

Comment:
Added a test for a bounding box in North America, South America, Madagascar and Japan.  I also put tests in to handle the 100k, 10k and 1k resolutions for those new bounding boxes.